apps/ui/src/app/playground/playground-client.tsx

```
@@ -21,23 +21,39 @@
~import { Alert, AlertDescription } from "@/lib/components/alert";
 import { SidebarProvider } from "@/lib/components/sidebar";
 import { useAppConfig } from "@/lib/config";
 import { useApi } from "@/lib/fetch-client";
+import { useRouter, useSearchParams } from "next/navigation";
```

ESLint

[error] `next/navigation` import should occur before import of `react`

(import/order)